koku/cost_models/cost_model_manager.py

```diff
@@ -0,0 +1,104 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Management layer for user defined rates."""
+
+import copy
+import logging
+
+from django.db import transaction
+
+from cost_models.models import CostModel, CostModelMap
+
+
+LOG = logging.getLogger(__name__)
+
+
+class CostModelManagerError(Exception):
```

Comment:
Why is this exception necessary?  I don't see anywhere that it gets raised.

Reply:
Good catch, removed it.